src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -1,21 +1,13 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { Graph } from 'components/Visualizations';
 import { getPlayerHistogram } from 'actions';
 import { playerHistogram } from 'reducers';
 import { withRouter } from 'react-router';
 import FlatButton from 'material-ui/FlatButton';
 import { deSnake } from 'utility';
-import histogramNames from './histogramNames';
+import { HistogramGraph } from 'components/Visualizations';
 import styles from './Histograms.css';
-
-
-const getAxis = (name, show = true) => ({
-  label: {
-    text: name,
-    show,
-  },
-});
+import histogramNames from './histogramNames';
```

Comment:
the histograms and trends will operate on the same fields.  You could probably genericize the name (numericalFields or something) and store them in a neutral location.